tests/test_prove_stat_validation.py

```
@@ -0,0 +1,124 @@
+from gradata._stats import trend_analysis
+
+ALPHA = 0.05
+N_TRIALS = 400     # 95% CI for p=0.05 at n=400 ≈ [0.031, 0.077]
+SERIES_LEN = 30    # realistic session count
+
+
+def _rejection_rate(series_factory, n_trials: int = N_TRIALS) -> float:
+    """Fraction of runs where Mann-Kendall rejects H0 at α=0.05."""
+    reject = 0
+    for i in range(n_trials):
+        rng = random.Random(1000 + i)
```

Ruff (0.15.10)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)